binsync/data/enum.py

```diff
@@ -52,6 +52,41 @@ def dump_many(cls, enums):
     def copy(self):
         return Enum(
             self.name,
-            self.value_map.copy(),
+            self.members.copy(),
             last_change=self.last_change
         )
+
+    def nonconflict_merge(self, enum2: "Enum"):
+        enum1 = self.copy()
+        if not enum2 or enum1 == enum2:
+            return enum1.copy()
+
+        constants = {
+            value for value in enum1.members.values()
+        }
+
+        for name, constant in enum2.members.items():
+            if name in enum1.members or constant in constants:
+                continue
+            enum1.members[name] = constant
+
+        return enum1
+
+    def conflict_merge(self, enum2: "Enum"):
```

Comment:
If you plan on keeping this here, add some comments describing what this does at a high level